pupil_src/shared_modules/pupil_recording/info/__init__.py

```diff
@@ -10,7 +10,9 @@
 """
 
 from .. import Version
+from .recording_info import RecordingInfo
 from .recording_info import RecordingInfoFile
+from .recording_info import RecordingInfoInvalidError
```

Comment:
these were needed when I was trying to move `pupil_recording` tests to the `tests/` directory. however, the tests no longer made sense, so I simply removed them. I will undo this change for now since it's not required.